crates/kitsune_p2p/kitsune_p2p/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- Gossip send failures and tgt expired events are now logged as warnings, rather than errors
```

Comment:
I can't help but suggest replacing the word 'tgt' with 'target' here.

Reply:
and possibly even a more clear word that situates what a "target" is

Reply:
updated to `tracing::warn!("Gossip partner has timed out, considering them expired for the current round {:?}", cert);`

how does that sound?